microservices/resource-management-control/src/service/service.js

```diff
@@ -1,54 +1,105 @@
-// const grpc = require('grpc');
+const grpc = require('grpc');
 
 const { getRepository } = require('typeorm');
```

Comment:
combine this imports const { getRepository, getConnection } = require('typeorm');